.github/workflows/cypress-firefox.yml

```
@@ -0,0 +1,22 @@
+name: Cypress push and pull
+
+on: [push, pull_request]
+
+jobs:
+  cypress-run:
+     name: firefox
+     runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: update test fixture
+        run: npm run pre:cy:genfix
+
+      - name: Cypress run
+        uses: cypress-io/github-action@v2
+        with:
+          build: npm run serve
```

```suggestion
          start: npm run serve
```
Same issue that Shaun pointed out.